src/App.jsx

```diff
@@ -3,13 +3,17 @@ import MenuHeader from './components/MenuHeader'
 import Footer from './components/Footer'
 import './index.css'
 import MainPageDisplay from './components/MainPageDisplay'
+import CreateFoodBagForm from './components/CreateFoodBagForm'
+import { useSelector } from 'react-redux'
 
 const App = () => {
+  const currentUser = useSelector(state => state.currentUser)
+
   return (
     <>
       <Suspense fallback={<h3>Loading...</h3>}>
         <MenuHeader />
-        <MainPageDisplay />
+        {currentUser ? <CreateFoodBagForm /> : <MainPageDisplay />}
```

Comment:
Why are you pointing directly to the foodbagform instead of the dashboard component?